solr/core/src/java/org/apache/solr/cli/HealthcheckTool.java

```diff
@@ -60,11 +60,11 @@ public List<Option> getOptions() {
         SolrCLI.OPTION_SOLRURL,
         SolrCLI.OPTION_ZKHOST,
         Option.builder("c")
-            .argName("COLLECTION")
+            .longOpt("name")
+            .argName("NAME")
             .hasArg()
             .required(false)
```

Comment:
[-1] AFAICT from [this bin/solr code](https://github.com/apache/solr/pull/1670/files#diff-49c29c8f653341f48008c38f5f2cf970fa430cdca29181c819d7bdcfcc722980L1003), the collection parameter should be required for the healthcheck tool.

Prior to this commit, we enforced this requirement in the bash/Windows-scripts, but now that we don't parse args or enforce required params there any more - we should update the `.required(...)` call here to be correct.

(Assuming I'm reading the old `bin/solr` code correctly at least; always possible I'm wrong.)

Reply:
https://issues.apache.org/jira/browse/SOLR-16882 for fixing this...   I will investigate the .bats test...